kafka/kafka-common/src/main/java/org/apache/apex/malhar/kafka/KafkaConsumerWrapper.java

```diff
@@ -230,17 +228,17 @@ public void run()
   }
 
   protected void handleNoOffsetForPartitionException(NoOffsetForPartitionException e,
-      KafkaConsumer<byte[], byte[]> consumer)
+      AbstractKafkaConsumer consumer)
   {
     // if initialOffset is set to EARLIST or LATEST
     // and the application is run as first time
     // then there is no existing committed offset and this error will be caught
     // we need to seek to either beginning or end of the partition
     // based on the initial offset setting
-    AbstractKafkaInputOperator.InitialOffset io =
```

Comment:
AbstractKafkaInputOperator spitted into 2 level hierarchy. AbstractBaseKafkaInputOperator consists of common design which is under kafka-common module and contains the createConsumer(...) abstract method. AbstractKafkaInputOperator extends from AbstractBaseKafkaInputOperator which is under malhar-kafka* modules.